style.css

```diff
@@ -214,6 +236,159 @@ h5 {
 	right: 0;
 }
 
+/* Profile page CSS */
+
+.profile-body-container {
+	display: grid;
+	grid-template-columns: 30% auto;
+	grid-column-gap: 1.5rem;
+	position: relative;
+}
+
+.grid-itemm {
+	width: 100%;
+	position: -webkit-sticky;
+  	position: sticky;
+  	top: 5rem;
+}
+
+#profile-box-grid {
+	display: grid;
+	width: 100%;
+	background-color: rgba(0, 0, 0, 0.5);
+	grid-template-columns: 40% auto;
+	border-radius: 10px;
+	padding: 1rem;
+	grid-column-gap: 1rem;
+}
+
+#profile-picture {
+	width: 7rem;
+	height: 7rem;
+	border-radius: 7rem;
+	object-fit: cover;
+	display: block;
+	margin-right: auto;
+	margin-left: auto;
+}
+
+#brief-profile-description {
+	width: 100%;
+	color: white;
+	margin: auto;
+}
+
+#profile-description {
+	grid-column-start: 1;
+	grid-column-end: 3;
+	line-height: 2rem;
+	color: white;
+	margin-top: 0.5rem;
+	margin-left: 0.5rem;
+}
+
+.achievement-box {
+	margin-left: auto;
+	margin-right: auto;
+	padding: 0.6rem;
+	display: grid;
+	width: 100%;
+	grid-template-columns: 25% auto;
+	grid-column-gap: 0.5rem;
+	margin-top: 1rem;
+	margin-bottom: 1rem;
+}
+
+#achievements-container {
+	width: 75%;
+	margin-left: auto;
+	margin-right: auto;
+	display: grid;
+	grid-row-gap: 0rem;
+	grid-template-columns: auto;
+}
+
+
+#achievements-container hr {
+	width: 100%;
+	height: 0;
+}
+
+.achievement-icon {
+	width: 5rem;
+	height: 5rem;
+	object-fit: cover;
+	margin: auto;
+	margin-top: 0;
+}
+
+.achievement-description {
+	margin: 0;
+	padding: 0;
+}
+
+.progress-bar {
+	width: 90%;
+	height: 1.5rem;
+	border: 2px solid #80d4ff;
+	border-radius: 1rem;
+	padding: 0;
+	position: relative;
+	margin-top: 1rem;
+}
+
+.percentage {
+	width: 100%;
+	position: absolute;
+	vertical-align: middle;
+	text-align: center;
+	height: 100%;
+	vertical-align: middle;
+	line-height: 1.5rem;
+}
+
+.bar100 {
+	border-radius: 2rem;
+	background-color: #80d4ff;
+	width: 100%;
+	height: 100%;
+}
+
+.bar50 {
+	border-radius: 2rem 0 0 2rem;
+	background-color: #80d4ff;
+	width: 50%;
+	height: 100%;
+}
+
+.bar25 {
+	border-radius: 2rem 0 0 2rem;
+	background-color: #80d4ff;
+	width: 25%;
+	height: 100%;
+}
+
+/* Font CSS classes */
+
+.big {
+	font-size: larger;
+}
+
+.bigger {
+	font-size: x-large;
+}
+
+.grey-box {
```

Comment:
These 2 classes don't seem to be related to fonts

Reply:
.big and .bigger can be used to replace the obsolete <big> html tag .

Reply:
.grey-box and .grid-container-two-columns